IMU/berryIMU.py

```diff
@@ -481,27 +469,21 @@ def signal_handler(sig, frame):
            # print("faceforward")
        # print(output)
         
-        '''
-        client.on_message = on_message
-        
-        client.connect_async("mqtt.eclipseprojects.io")
         
-        client.loop_start()
-        '''
         if lastmove != output:
-            #for i in range(10):
-            client.publish("topic/movement", output, qos=1)
+            client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
Do you think it would make sense to instantiate/close the connection outside the loop and just send the data here? I was thinking that it could potentially improve latency bc you wouldn't have to initialize and close the connection every time you'd have to send data.

Reply:
Same thing for the movenet changes, I was thinking you could make the client a private variable in the constructor instead of the IP address

Reply:
Just tested with movenet and the latency seems fine. I'm wondering if anything improves for the IMU since if not, having to manually enter IP addresses might be a hurdle.